test/spacebinding/spacebinding.go

```diff
@@ -0,0 +1,25 @@
+package spacebinding
+
+import (
+	"github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewSpaceBinding(name, mur, space, spaceRole string) *v1alpha1.SpaceBinding {
```

Comment:
yeah, we need a naming pattern for the SpaceBinding which will most likely be `<mur>-<space>`, but that's important for their creation in production. Unit tests shouldn't care about the name - it's still just a name. 

But I understand what you meant and it's a good point. I changed the function to avoid any confusion: 04cb8367